internal/k8s/main.go

```diff
@@ -42,6 +42,7 @@ func ParseNameNS(input string) (string, string, error) {
 func GetNodeIPOrName(kubeClient clientset.Interface, name string, useInternalIP bool) string {
 	node, err := kubeClient.CoreV1().Nodes().Get(name, metav1.GetOptions{})
 	if err != nil {
+		glog.Error(err)
```

Comment:
I'd prepend some context like you did in `internal/file/file.go`. Eg. `"Error getting node %s: %s" , name, err`

Reply:
Thanks will do!